docs/package/gluon-harden-dropbear.rst

```diff
@@ -0,0 +1,31 @@
+gluon-harden-dropbear
+=====================
+
+This package reduces the attack surface of dropbear, the SSH server on the router.
+
+If the root account either has no password configured or is locked,
+password-authorization is disabled in dropbear's settings.
+
+
+If furthermore no SSH key is authorized to login, the ``dropbear`` service is disabled.
+
+Changing the password or updating authorized keys
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Via console
+"""""""""""
+
+Upon editing */etc/dropbear/authorized_keys* or changing root's password,
+a call to *gluon-reconfigure* as well as a reboot might be needed in order to have dropbear launched conditionally upon boot.
+
+.. code-block:: bash
+
+  passwd
+  gluon-reconfigure
+  reboot
+
+
+In setup-mode
+"""""""""""""
+
+As *gluon-reconfigure* and a reboot happen upon leaving the setup-mode, no further steps are required.
```

Comment:
This is not entirely correct - `gluon-reconfigure` only happens when rebooting using the `Save and reboot` button in the web UI - not when switching from setup mode to regular operating by rebooting via SSH.

Reply:
Even if it's not supported officially, quickly preparing multiple nodes over SSH (set password, edit gluon-setup-mode UCI, reboot) is likely to be done occasionally, and it may result in confusion if the nodes are not actually reachable via SSH until the next upgrade. Personally, I almost never use the Web UI during testing, and instead use the above steps to set up my nodes.

Reply:
How about the following?
```suggestion
As *gluon-reconfigure* is run when rebooting from  the setup-mode web interface, no further steps are required.
```

Reply:
resolved in 4b343b7c0ab94ce2395aa7649d7e65a06817dc33